cmd/check-gh-automation/main.go

```diff
@@ -157,6 +157,18 @@ func checkRepos(repos []string, bots []string, ignore sets.Set[string], client a
 			}
 		}
 
+		// Add a check for openshift-cherrypick-robot
+		isCherryPickRobotMember, err := client.IsMember(org, "openshift-cherrypick-robot")
```

Comment:
The `openshift-cherrypick-robot` does not need to be a member in every org, only those with the `cherrypick` plugin enabled. You will need to check the prow configs for the org to see if the plugin is enabled prior to checking for membership